packages/bbui/src/Tabs/Tab.svelte

```diff
@@ -27,14 +27,30 @@
     setTabInfo()
   })
 
+  //Ensure that the underline is in the correct location
+  $: {
+    if ($selected.title === title && tab_internal) {
+      if ($selected.info?.left !== tab_internal.getBoundingClientRect().left) {
+        $selected = {
+          ...$selected,
+          info: tab_internal.getBoundingClientRect(),
+        }
+      }
+    }
+  }
```

Comment:
Yep, exactly. When you click on a tab, the underline offset is automatically updated in the Tabs component (calculateIndicatorOffset). If you wish to navigate to another tab without clicking on one directly, you have to trigger the recalculation of the tab underline manually. 

If you click on the "App Version" card header, it will update the selected tab key only. At that point, the selected tab will determine that it is now the current tab and ensure that its own position be pushed into the 'tab' context. Updating the context in this way triggers the repositioning in the Tabs parent component with the correct data.  Hopefully than makes sense!